aws/tailers/stack_event.go

```diff
@@ -38,14 +68,21 @@ func (t *stackEventTailer) Tail(w io.Writer) error {
 	}
 
 	if t.pollingFrequency < 5*time.Second {
-		return fmt.Errorf("invalid polling frequency: %s", t.pollingFrequency)
+		return fmt.Errorf("invalid polling frequency: %s, must be greater than 5s", t.pollingFrequency)
 	}
 
+	// create new tabwriter and
+	// add header based on filters
```

Comment:
Unecessary comments: first is redundant with `tabwriter.NewWriter()`, and second one is not necessary.